packages/legend-studio/src/models/protocols/pure/v1/V1_PureGraphManager.ts

```diff
@@ -1944,62 +1945,51 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
 
   // --------------------------------------------- V1_Store ---------------------------------------------
 
-  generateStore = flow(function* (
+  buildDatabase = flow(function* (
     this: V1_PureGraphManager,
-    input: GenerateStoreInput,
-  ): GeneratorFn<string> {
-    const generateStoreInput = new V1_GenerateStoreInput();
-    generateStoreInput.connection = V1_transformConnection(
+    input: DatabaseBuilderInput,
+  ): GeneratorFn<Entity[]> {
+    const graph = (yield this.buildPMCDFromDatabaseBuilderInput(
+      input,
+    )) as V1_PureModelContextData;
+    return this.pureModelContextDataToEntities(graph);
+  });
+
+  private buildPMCDFromDatabaseBuilderInput = flow(function* (
```

Comment:
I think we should name this properly, if not, since this is `private` and `used` only in one place, we might as well move this into the previous function